src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -287,6 +288,11 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
                          .GetSection(ConfigurationSectionNames.Scale)
                          .Bind(o);
                     });
+                services.AddOptions<WorkerConcurrencyOptions>()
```

Comment:
Doesn't just services.AddOptions<WorkerConcurrencyOptions>() work? Shouldn't need the separate .Configure call I believe. Just let DI create the instance.